clang/lib/Sema/SemaDecl.cpp

```diff
@@ -10333,6 +10334,74 @@ Sema::ActOnFunctionDeclarator(Scope *S, Declarator &D, DeclContext *DC,
       }
     }
 
+    // Loop over the parameters to see if any of the size expressions contains
```

Comment:
Can we
 - Put all of that in a separate function
 - Maybe not run it in modes VLAs are not supported